### PR TITLE
fix(tile-converter): black models issue

### PR DESCRIPTION
**Pull request by belom88**

### CODE REVIEW

File: modules/tile-converter/src/i3s-converter/helpers/geometry-converter.js [hunk 1]
```diff
@@ -375,6 +362,7 @@ function transformVertexNormals(vertexVector, calleeArgs) {
 function flattenTexCoords(texCoords, indices) {
   const newTexCoords = new Float32Array(indices.length * VALUES_PER_TEX_COORD);
   if (!texCoords) {
+    newTexCoords.fill(1);
```
Comment (ibgreen) [inline modules/tile-converter/src/i3s-converter/helpers/geometry-converter.js:365]: A bit strange to have uv coordinates if there are no textures... but ok. 

Explain in comment that we are returning dummy UVs?

File: modules/tile-converter/src/i3s-converter/helpers/geometry-converter.js [hunk 2]
```diff
@@ -396,6 +384,7 @@ function flattenColors(colorsAttribute, indices) {
   const components = colorsAttribute?.components || VALUES_PER_COLOR_ELEMENT;
   const newColors = new Uint8Array(indices.length * components);
   if (!colorsAttribute) {
+    newColors.fill(255);
```
Comment (ibgreen) [inline modules/tile-converter/src/i3s-converter/helpers/geometry-converter.js:387]: Explain in comment that this is intended to be "all white" by default

**Review by ibgreen — APPROVED** [review]
If this was a bug, maybe add a test case that fails before the change and then is fixed by this change?

File: modules/tile-converter/src/i3s-converter/helpers/geometry-converter.js (hunk 1 above)
Comment (belom88) [reply modules/tile-converter/src/i3s-converter/helpers/geometry-converter.js:367]: Done

File: modules/tile-converter/src/i3s-converter/helpers/geometry-converter.js (hunk 2 above)
Comment (belom88) [reply modules/tile-converter/src/i3s-converter/helpers/geometry-converter.js:390]: Done

**Pull request merged**